tests/Doctrine/Tests/DBAL/Schema/SchemaTest.php

```diff
@@ -392,7 +396,7 @@ public function testVisitsVisitor()
         $visitor->expects($this->exactly(2))
             ->method('acceptSequence');
 
-        self::assertNull($schema->visit($visitor));
+        $this->assertNull($schema->visit($visitor));
```

Comment:
this changed sneaked in probably accidentally

Reply:
updated